content_sync/migrations/0002_backpopulate_content_sync_state.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.1.6 on 2021-04-15 19:13
+import json
+from hashlib import sha256
+
+from django.db import migrations
+
+
+def backpopulate_content_sync_states(apps, schema_editor):
+    """ Backpopulate ContentSyncState for existing WebsiteContent records """
+    WebsiteContent = apps.get_model("websites", "WebsiteContent")
+    ContentSyncState = apps.get_model("content_sync", "ContentSyncState")
+
+    for content in WebsiteContent.objects.filter(content_sync_state=None):
```

Comment:
There's about 88K of these on RC, I've got about the same amount in my local instance and docker memory usage went pretty high (~2.5 GB), wondering if that might cause problems on heroku.  One thing I did in a similar migration for open was use a chunked iterator:

```
def chunked_iterator(queryset):
    paginator = Paginator(queryset, 100)
    for page in range(1, paginator.num_pages + 1):
        for obj in paginator.page(page).object_list:
            yield obj
            
 for content in chunked_iterator(WebsiteContent.objects.filter(content_sync_state=None)):
 ```
 
Or can leave as is and see how it goes, maybe it's not necessary.